packages/protocol/contracts/governance/BlockchainParameters.sol

```diff
@@ -16,7 +16,18 @@ contract BlockchainParameters is Ownable, Initializable {
 
   ClientVersion private minimumClientVersion;
 
+  uint256 public gasForDebitFromTransactions;
+  uint256 public gasForCreditToTransactions;
+  uint256 public gasToReadErc20Balance;
+  uint256 public gasToReadTobinTax;
+  uint256 public gasForNonGoldCurrencies;
```

Comment:
Isn't this a function of `creditTo`, `debitFrom`, and `readErc20`? Could either this one, or those three be removed?

Reply:
Yeah, currently it would make more sense to remove the other three.